.rustfmt.toml

```diff
@@ -1,67 +1,74 @@
-max_width = 90 # changed
-hard_tabs = false
-tab_spaces = 4
-newline_style = "Auto"
-use_small_heuristics = "Default"
-indent_style = "Block"
-wrap_comments = false
-format_code_in_doc_comments = false
-comment_width = 80
-normalize_comments = true # changed
-normalize_doc_attributes = false
-# license_template_path = "LICENSE_TEMPLATE" # changed
-format_strings = false
-format_macro_matchers = false
-format_macro_bodies = true
+# Most of the changed settings had defaults made for backcompat with earlier editions.
+# Brief descriptions are provided for nontrivial changes, otherwise a simple "changed" is
+# used.
+
+# `rustfmt` internal settings
+#
+# emit_mode = "Files"
+# make_backup = false
+
+binop_separator = "Front"
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 1
+brace_style = "SameLineWhere"
+color = "Auto"
+combine_control_expr = false # better all around in our opinions
+comment_width = 80 # changed
+condense_wildcard_suffixes = true # changed
+control_brace_style = "AlwaysSameLine"
+disable_all_formatting = false
+edition = "2018" # changed
 empty_item_single_line = true
-struct_lit_single_line = true
+enum_discrim_align_threshold = 0
+error_on_line_overflow = true # be more strict
+error_on_unformatted = true # be more strict
+fn_args_layout = "Tall"
+# unchanged, this does not work well with the long expressions of this crate
 fn_single_line = false
-where_single_line = false
+force_explicit_abi = true
+force_multiline_blocks = true # help certain closures
+format_code_in_doc_comments = true # changed
+format_macro_bodies = true # changed
+format_macro_matchers = true # changed
+format_strings = true # changed
+hard_tabs = false
+hide_parse_errors = false
+ignore = []
 imports_indent = "Block"
-imports_layout = "Vertical" # changed
+imports_layout = "Vertical" # makes editing quicker
+indent_style = "Block"
+inline_attribute_width = 0 # unchanged, for ease of editing
```

Comment:
This option is unstable and since we only set it to its default pls remove.

Reply:
ok